apps/vr-tests/package.json

```diff
@@ -9,9 +9,8 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "screener": "screener-storybook --conf screener.config.js --debug",
     "screener:build": "cross-env NODE_OPTIONS=--max-old-space-size=3072 just-scripts storybook:build",
-    "screener:local": "screener-storybook --conf screener.local.config.js --debug",
```

Comment:
As discussed offline with build team, we are OK to remove local screener runs

Reply:
Can we remove screener.local.config.js in this PR too? Isn't needed anymore if we're dropping local Screener testing.